test/lib/mayaUsd/fileio/testEditAsMaya.py

```diff
@@ -65,7 +64,6 @@ def tearDownClass(cls):
     def setUp(self):
         cmds.file(new=True, force=True)
 
-    @unittest.skipUnless(ufeFeatureSetVersion() >= 3, 'Test only available in UFE v3 or greater.')
```

Comment:
The CMakeLists.txt file explicitly adds this test file only for UFE v3, so further checking in the test is redundant.